backend/pom.xml

```diff
@@ -52,6 +52,12 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>dev.langchain4j</groupId>
```

Comment:
WHy do you need that?

Reply:
Because I had exception during runtime with Antrhopic. I changed the version of dependency anthropic and it is no needed more to have this general dependency